shared/common/_channel-management-api.mdx

```diff
@@ -16,7 +16,7 @@ The `Content-Type` field in all HTTP request headers is `application/json`. All
 
 ### Authentication
 
-The Agora Channel Management RESTful APIs only support HTTPS. Before sending HTTP requests, you must generate a Base64-encoded credential with the **Customer ID** and **Customer Secret** provided by Agora, and pass the credential to the `Authorization` field in the HTTP request header. See [HTTP authentication](./restful-authentication).
+The Agora Channel Management RESTful APIs only support HTTPS. Before sending HTTP requests, you must generate a Base64-encoded credential with the **Customer ID** and **Customer Secret** provided by Agora, and pass the credential to the `Authorization` field in the HTTP request header. See [HTTP authentication](https://docs.agora.io/en/video-calling/reference/restful-authentication).
```

Comment:
No FQDN links please. 

This is a shared page so you need to do a relative link and conditional text. 

Same for the rest of the docs.